templates/cluster/azure-hosted-cp/templates/k0smotroncontrolplane.yaml

```diff
@@ -25,19 +25,28 @@ spec:
       extensions:
         helm:
           repositories:
-            - name: mirantis
+            - name: cloud-provider-azure
```

Comment:
What if the external repository gets changed and it no longer represents `mirantis`, at a minimum it's still representing the `cloud-provider-azure` repo in both instances of change or no change without the need for if/else logic.

Reply:
This external repository will not get changed unless we will change it. Right now it's a repo with random forked charts.

It's also more clear when you check the k0s' `chart` object you immediately know that it's a forked chart in our repo and not an upstream one.

Reply:
I mean the external repository _will get changed_ when someone specifically changes it, when they override the repository with their own value. We're allowing users to modify the external charts repository to whatever value they choose.  

Honestly is arbitrary, it's a name, but I felt it made more sense to make it a generic name then `mirantis` since if the external repository name changes it no longer potentially represents the mirantis repository, it represents the repository for `cloud-provider-azure`.

Reply:
Ok, let's leave it. I don't see a point of blocking this PR on these